README.md

```diff
@@ -129,6 +162,7 @@ Then, to run the app locally,
 ```
 $ bundle install
 $ yarn install
+$ docker-compose up -d (if using docker)
```

Comment:
Maybe add a `#` before the comment so that if someone copies and pastes the whole line, it will still work?